katdal/test/test_applycal.py

```diff
@@ -345,40 +331,42 @@ def test_add_sensors_does_nothing_if_no_ants_pols_or_spw(self):
         n_virtuals_after = len(cache.virtual)
         assert_equal(n_virtuals_after, n_virtuals_before)
         attrs = ATTRS.copy()
-        del attrs['cal_center_freq']
+        del attrs['center_freq']
         add_applycal_sensors(self.cache, attrs, FREQS)
         n_virtuals_after = len(cache.virtual)
         assert_equal(n_virtuals_after, n_virtuals_before)
 
     def test_delay_sensors(self):
         for n, ant in enumerate(ANTS):
             for m, pol in enumerate(POLS):
-                sensor_name = 'Calibration/{}{}_correction_K'.format(ant, pol)
+                sensor_name = 'Calibration/Corrections/cal/K/{}{}'.format(ant, pol)
                 sensor = self.cache.get(sensor_name)
                 assert_array_equal(sensor[10 + n], delay_corrections(m, n))
 
     def test_bandpass_sensors(self):
         for n, ant in enumerate(ANTS):
             for m, pol in enumerate(POLS):
-                sensor_name = 'Calibration/{}{}_correction_B'.format(ant, pol)
+                sensor_name = 'Calibration/Corrections/cal/B/{}{}'.format(ant, pol)
                 sensor = self.cache.get(sensor_name)
                 assert_array_equal(sensor[12 + n], bandpass_corrections(m, n))
 
     def test_gain_sensors(self):
         for n, ant in enumerate(ANTS):
             for m, pol in enumerate(POLS):
-                sensor_name = 'Calibration/{}{}_correction_G'.format(ant, pol)
+                sensor_name = 'Calibration/Corrections/cal/G/{}{}'.format(ant, pol)
                 sensor = self.cache.get(sensor_name)
                 assert_array_equal(sensor[:], gain_corrections(m, n))
 
     def test_unknown_inputs_and_products(self):
-        known_input = 'Calibration/{}{}'.format(ANTS[0], POLS[0])
+        known_input = '{}{}'.format(ANTS[0], POLS[0])
         with assert_raises(KeyError):
-            self.cache.get('Calibration/unknown_correction_K')
+            self.cache.get('Calibration/Corrections/cal/K/unknown')
         with assert_raises(KeyError):
-            self.cache.get(known_input + '_correction_unknown')
+            self.cache.get('Calibration/Corrections/cal/unknown/' + known_input)
         with assert_raises(KeyError):
-            self.cache.get(known_input + '_correction_K_unknown')
+            self.cache.get('Calibration/Corrections/cal/K_unknown/' + known_input)
+        with assert_raises(KeyError):
+            self.cache.get('Calibration/Corrections/unknown/K/' + known_input)
```

Comment:
Should this test be extended to test the Calibration/Product namespace too?

Reply:
Good idea.